src/core/server/integration_tests/saved_objects/migrations/group3/deferred_migrations.test.ts

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
```

Comment:
We have 12 tests running in group3 already. I recommend this and any other new ones be in a new group4.
It's going to make resolving CI failures for new implementations easier and faster to identify.